tests/internal/scripts/ccuda/cuda_grid.py

```diff
@@ -24,7 +24,7 @@ def func_3d(a):
     n_blocks = 1
     arr = cuda.array([0, 1, 2, 3, 4])
     cuda.deviceSynchronize()
-    # Sice we dont support multi-dim n_block / threads_per_block
-    # func_2d and func_3d won't compile
     func_1d[n_blocks, threads_per_block](arr)
+    func_2d[n_blocks, threads_per_block](arr)
```

Comment:
after the discussion that we had in the meeting I thought that we agreed to just use the `func_1d` for now until we add the support for multi-dim `n_block`  and `threads_per_block`.

Reply:
I probably added it before our meeting, i'll revert it.